frontend/app/work_packages/helpers/api-helper.js

```diff
@@ -51,14 +43,16 @@ module.exports = function() {
           messages.push(response.message);
         }
 
-        message = messages.join(' ');
-
-        return message;
+        return messages;
       }
     },
 
+    getErrorMessage: function(error) {
+      return ApiHelper.getErrorMessages(error).join(' ');
```

Comment:
First of all: Thank you for introducing `getErrorMessages` :+1: 

But secondly: Who would ever need error messages joined by a single space character? This sounds like the least displayable error string imaginable (except for the same string reversed, maybe).

Okay, I digged a little. The only caller seems to be [here](https://github.com/opf/openproject/blob/4889b8615c51e877b19a5a066f04cdebf494819b/frontend/app/work_packages/directives/inplace_editor/inplace-editor-edit-pane-directive.js#L160). Then it is used downstream [here](https://github.com/opf/openproject/blob/4889b8615c51e877b19a5a066f04cdebf494819b/frontend/app/work_packages/directives/inplace_editor/inplace-editor-edit-pane-directive.js#L45).

The only reason that this did not produce ugly errors yet, is that when there is a general (`_common`) error, the API will AFAIK only ever return a single error. And apparently there are no (observed) cases of more than one error per field, because that would also blow up.